tests/integration/dart/test/wallets_client_test.dart

```diff
@@ -53,6 +53,18 @@ void main() {
       expect(createdWallet.wallet!.keys!.length, greaterThan(0));
       expect(createdWallet.wallet!.keys!.first.id, isNotEmpty);
       expect(createdWallet.wallet!.keys!.first.ari, isNotEmpty);
+
+      // delete newly created ^^ test wallet
+      final walletId = createdWallet.wallet?.id ?? '';
```

Comment:
similar question here as above.
Does it sometimes happen that the walletId is empty?
Moreover, in this test, if createdWallet always exist, there could perhaps be a way os telling this to the dart static type checker?

Reply:
code suggestions are very welcomed :)

Reply:
This PR's goal was to make tests pass.